### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by HKABIG**
Fixed some Typos

### CODE REVIEW

**Comment by HKABIG** [comment]
@rkrasiuk please review the changes

File: crates/revm/CHANGELOG.md [hunk 1]
```diff
@@ -261,7 +261,7 @@ Github Changelog:
 * de83db6 - fix: feature flags (#330) (2 weeks ago) <Wodann>
 * b60269c - `revm`: mark `with-serde` feature as deprecated (#328) (2 weeks ago) <Enrique Ortiz>
 * 63bf475 - make load_account pub (#325) (3 weeks ago) <rakita>
-* 0ef0197 - Cleanup, move hot fields toggether in Interpreter (#321) (3 weeks ago) <rakita>
+* 0ef0197 - Clean-up, move hot fields together in Interpreter (#321) (3 weeks ago) <rakita>
```
Comment (rakita) [inline crates/revm/CHANGELOG.md:264]: This is git commit, should not be changed

File: crates/revm/CHANGELOG.md [hunk 2]
```diff
@@ -249,7 +249,7 @@ Github Changelog:
 * dc9818f - (HEAD -> o/bump, origin/bump_v20) Bump v20 (13 hours ago) <rakita>
 * 75ef0f1 - (origin/main, origin/HEAD) feat: Staticcall internal return (#349) (13 hours ago) <rakita>
 * 0194b37 - (t) fix bug introduced in last commit (13 hours ago) <rakita>
-* 7b00f32 - Cleanup imports (#348) (14 hours ago) <rakita>
+* 7b00f32 - Clean-up imports (#348) (14 hours ago) <rakita>
```
Comment (rakita) [inline crates/revm/CHANGELOG.md:252]: This is git commit, should not be changed

File: crates/interpreter/CHANGELOG.md [hunk 3]
```diff
@@ -29,8 +29,8 @@ Full git log:
 * 8206193 - feat: add "kzg" as a separate feature (#746) (2 hours ago) <DaniPopes>
 * 516f62c - perf(interpreter): remove dynamic dispatch from all instructions (#739) (5 days ago) <DaniPopes>
 * 26af13e - EIP-7516: BLOBBASEFEE opcode (#721) (5 days ago) <rakita>
-* 36e71fc - fix: dont override instruction result (#736) (6 days ago) <rakita>
-* d926728 - perf: refactor interpreter internals and cleanup (#582) (6 days ago) <DaniPopes>
+* 36e71fc - fix: don't override instruction result (#736) (6 days ago) <rakita>
+* d926728 - perf: refactor interpreter internals and clean-up (#582) (6 days ago) <DaniPopes>
```
Comment (rakita) [inline crates/interpreter/CHANGELOG.md:33]: This is git commit, should not be changed

File: documentation/src/crates/revm/journaled_state.md [hunk 4]
```diff
@@ -136,6 +136,6 @@ This methodology allows Ethereum to maintain an internal record of accessed acco
 
 #### Rationale
 
-- Security: Previously, these opcodes were underpriced, making them susceptible to DoS attacks where an attacker would simply send transactions that access or call a large number of accounts. By increasing the gas costs, the EIP intends to mitigate these potential security risks.
+- Security: Previously, these opcodes were under-priced, making them susceptible to DoS attacks where an attacker would simply send transactions that access or call a large number of accounts. By increasing the gas costs, the EIP intends to mitigate these potential security risks.
```
Comment (rakita) [inline documentation/src/crates/revm/journaled_state.md:139]: Isn't this one word?

File: CHANGELOG.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ Migration to alloy primitive types.
 # v25 tag
 date: 28.09.2023
 
-Bigger release. Cancun support, revm State added and some cleanup refactoring.
+Bigger release. Cancun support, revm State added and some clean-up refactoring.
```
Comment (rakita) [inline CHANGELOG.md:16]: Isn't cleanup one word?

**Review by rakita — CHANGES_REQUESTED** [review]
left nits

File: crates/revm/CHANGELOG.md (hunk 1 above)
Comment (HKABIG) [reply crates/revm/CHANGELOG.md:264]: yes you are right I will update my pr

File: crates/revm/CHANGELOG.md (hunk 2 above)
Comment (HKABIG) [reply crates/revm/CHANGELOG.md:252]: ok updating the pr

File: documentation/src/crates/primitives/specifications.md [hunk 6]
```diff
@@ -10,6 +10,6 @@ The `enabled()` method in `SpecId` is used to check if one spec is enabled on an
 
 The `Spec` trait is used to abstract the process of checking whether a given spec is enabled. It only has one method, `enabled()`, and a constant `SPEC_ID`.
 
-The module then defines various `Spec` structs, each representing a different hard fork. These structs implement the `Spec` trait and each struct's `SPEC_ID` corresponds to the correct `SpecId` variant.
+The module then defines various `Spec` structs, each representing a different hard fork. These structs implement the `Spec` trait and each strut's `SPEC_ID` corresponds to the correct `SpecId` variant.
```
[the remark this answers is not in the document]
Comment (HKABIG) [reply documentation/src/crates/primitives/specifications.md:13]: ok I will update this too

File: documentation/src/crates/revm/journaled_state.md (hunk 4 above)
Comment (HKABIG) [reply documentation/src/crates/revm/journaled_state.md:139]: actually with hyphen it can be looked upon as  a compound adjective making it more readable

File: CHANGELOG.md (hunk 5 above)
Comment (HKABIG) [reply CHANGELOG.md:16]: Yes you are right let me update this

**Comment by HKABIG** [comment]
@rakita I have committed changes according to your requested changes

**Comment by HKABIG** [comment]
@rakita can you review if possible by today

File: crates/interpreter/CHANGELOG.md [hunk 7]
```diff
@@ -89,7 +89,7 @@ Changelog:
 * d0038e3 - chore(deps): bump arbitrary from 1.2.3 to 1.3.0 (#428) (2 weeks ago) <dependabot[bot]>
 * dd0e227 - feat: Add all internals results to Halt (#413) (4 weeks ago) <rakita>
 * d8dc652 - fix(interpreter): halt on CreateInitcodeSizeLimit (#412) (4 weeks ago) <Roman Krasiuk>
-* a193d79 - chore: enabled primtive default feature in precompile (#409) (4 weeks ago) <Matthias Seitz>
+* a193d79 - chore: enabled primitive default feature in precompile (#409) (4 weeks ago) <Matthias Seitz>
```
Comment (rakita) [inline crates/interpreter/CHANGELOG.md:92]: revert

File: crates/revm/CHANGELOG.md [hunk 8]
```diff
@@ -36,7 +36,7 @@ Full git log:
 * f72eaa0 - chore: error type for block header (#731) (5 days ago) <hack3r-0m>
 * cb39117 - fix(eip4844): Pass eth tests, additional conditions added. (#735) (6 days ago) <rakita>
 * c2cde03 - fix: use CANCUN precompile id for CANCUN SpecId (#733) (6 days ago) <Dan Cline>
-* d926728 - perf: refactor interpreter internals and cleanup (#582) (6 days ago) <DaniPopes>
+* d926728 - perf: refactor interpreter internals and clean-up (#582) (6 days ago) <DaniPopes>
```
Comment (rakita) [inline crates/revm/CHANGELOG.md:39]: revert

File: crates/revm/CHANGELOG.md [hunk 9]
```diff
@@ -330,7 +330,7 @@ Main changes:
 * JournaledState (#175)
     * Optimize handling of precompiles. Initialization and account loading.
     * Fixes SELFDESTRUCT bug.
-* Optimize calldataload. Some cleanup (#168)
+* Optimize calldataload. Some clean-up (#168)
```
Comment (rakita) [inline crates/revm/CHANGELOG.md:333]: cleanup

File: crates/revm/CHANGELOG.md [hunk 10]
```diff
@@ -462,10 +462,10 @@ Changes:
 * optimize i256
 * switch stacks from H256 with U256
 * Error's refactor to Return
-* clippy/warnings/fmt cleanup
+* clippy/warnings/fmt clean-up
```
Comment (rakita) [inline crates/revm/CHANGELOG.md:465]: cleanup

File: crates/revm/CHANGELOG.md [hunk 11]
```diff
@@ -432,8 +432,8 @@ There is bug introduced in last release with gas blcok optimization, it will cra
 
 Changes:
 * Bug fix for unknown OpCode
-* Omit edgecase high nonce test. tracer gas fix 
-* Some internal cleanup
+* Omit edge case high nonce test. tracer gas fix 
+* Some internal clean-up
```
Comment (rakita) [inline crates/revm/CHANGELOG.md:436]: cleanup

File: crates/revm/CHANGELOG.md [hunk 12]
```diff
@@ -486,10 +486,10 @@ date: 27.10.2021
 Interface revamped and now looks a lot better.
 
 Log:
-* 1b1ebd8 - [revm] Interface. Inspector added, Env cleanup. revm-test passes (9 hours ago) <rakita>
+* 1b1ebd8 - [revm] Interface. Inspector added, Env clean-up. revm-test passes (9 hours ago) <rakita>
```
Comment (rakita) [inline crates/revm/CHANGELOG.md:489]: revert

File: crates/revm/CHANGELOG.md [hunk 13]
```diff
@@ -486,10 +486,10 @@ date: 27.10.2021
...
 * 351d4e0 - BIG interface change (11 hours ago) <rakita>
 * a723827 - no_sdt to no_std (2 days ago) <rakita>
-* a449bed - [precompiles] spelling, small cleanup (2 days ago) <rakita>
+* a449bed - [precompiles] spelling, small clean-up (2 days ago) <rakita>
```
Comment (rakita) [inline crates/revm/CHANGELOG.md:492]: revert

File: crates/revm/CHANGELOG.md [hunk 14]
```diff
@@ -498,7 +498,7 @@ Same as v0.2.1 but added readme.
 # v0.2.1
 date: 25.10.2021
 
-Big refactor, cleanup changes, and updating tests. EIP-3607 added.
+Big refactor, clean-up changes, and updating tests. EIP-3607 added.
```
Comment (rakita) [inline crates/revm/CHANGELOG.md:501]: cleanup
